### DISCUSSION TITLE
SAVE WPF THEME SETTING ON CLOSING THE APP

### DISCUSSION THREAD

**Discussion by d202021** [MahApps/MahApps.Metro#4243] [Q&A] [answered]
How can I save the theme setting applied by user, such that the next time the app is run, it runs with the theme settings applied by the user previously before closing the app.

**Comment by fredericoregateiro** [comment]
There are several ways to save and reload the selected theme, you can check the docs:
[https://mahapps.com/docs/themes/thememanager](https://mahapps.com/docs/themes/thememanager)

**Reply by punker76** [reply]
https://mahapps.github.io/mahapps.com/docs/themes/thememanager
